extensions/jdbc/jdbc-h2/runtime/pom.xml

```diff
@@ -13,6 +13,14 @@
     <name>Quarkus - JDBC - H2 - Runtime</name>
     <description>Connect to the H2 database via JDBC</description>
     <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-datasource</artifactId>
```

Comment:
@stuartwdouglas I haven't yet looked deeper but just as a quick fix to make the enhanced extension dependency validation to pass I had to add `quarkus-datasource` extension as a runtime dependency to most of the jdbc extensions. Otherwise, it would complain that the deployment artifact depends on `quarkus-datasource-deployment` through `quarkus-devservices-XXX` but the `quarkus-datasource` isn't found among the runtime dependencies.